### PR TITLE
EASYOPAC-1158 - Temporary Bibliofil commit.

### PR DESCRIPTION
**Pull request by mihaelad**
#### Link to issue

https://inlead.atlassian.net/browse/EASYOPAC-1158

#### Description

Adaptation for bibliofil theme 

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 1]
```diff
@@ -0,0 +1,438 @@
+//
+// Styling of site header
+
+//@import '../base.scss';
+
+//Overid the background for body
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot. Override is spelled with "e" in the end.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+  color: $white;
+}
+
+h1,
+.node-ding-news.node-full h1,
+.node-ding-event.node-full h1 {
+  color: $white;
+}
+
+//Overid the background for content and messages
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot. Override is spelled with "e" in the end.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+ div.panels-ipe-placeholder {
+    background-color: $bibliofil-dark;
+    background-image: none;
+  }
+
+.content-wrapper {
+  .pane-content {
+    .frontpage-layout {
+      .panel-pane{
+    background-color: $bibliofil;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Indent is wrong.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+  }
+
+  .panel-pane.system-messages,
+  .panel-pane .messages.error {
+    background-color: $bibliofil-dark;
+    background-image: none;
+  }
+}
+ div.panels-ipe-placeholder {
+    background-color: $bibliofil-dark;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Indent should be 2 spaces.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+      .panel-pane{
+    background-color: $bibliofil;
+    background-image: none;
+
+      }
+    }
+  }
+}
+
+//Overid the background and color for input
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot. Override is spelled with "e" in the end.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+.panel-pane.pane-import {
+  input[type=text] {
+    background-color: $white;
+    color: $bibliofil-button;
+  }
+}
+
+//Overid event and news background
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot. Override is spelled with "e" in the end.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 7]
```diff
@@ -0,0 +1,438 @@
...
+
+.node-ding-news.node-full,
+.node-ding-event.node-full,
+.pane-views-panes.pane-ding-library-ding-library-libraries-list {
+  background-color: $bibliofil;
+  background-image: none;
+}
+
+//Overid the background and color for input
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot. Override is spelled with "e" in the end.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 8]
```diff
@@ -0,0 +1,438 @@
...
+
+.pane-views-exp-ding-event-ding-event-list .form-item input {
+  background-color: $white;
+  color:  $text-input;
+}
+
+//Overid the background for submint button
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot. Override is spelled with "e" in the end. Submit is misspelled.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 9]
```diff
@@ -0,0 +1,438 @@
...
+
+input[type="submit"] {
+  background-color: $bibliofil-button;
+}
+
+//table
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 10]
```diff
@@ -0,0 +1,438 @@
...
+  &:nth-child(2n + 1) {
+    background-color: $bibliofil-dark;
+  }
+}
+
+tr.odd {
+  background-color: $bibliofil-dark;
+}
+
+// More link button
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 11]
```diff
@@ -0,0 +1,438 @@
...
+
+//table
+tr {
+  &:nth-child(2n + 1) {
+    background-color: $bibliofil-dark;
+  }
+}
+
+tr.odd {
+  background-color: $bibliofil-dark;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Any reason why we don't merge all selectors that define this property?

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 12]
```diff
@@ -0,0 +1,438 @@
...
+}
+
+// More link button
+//	//
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Remove.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 13]
```diff
@@ -0,0 +1,438 @@
...
+.more-link {
+  a {
+    color: $charcoal-opacity-dark;
+  }
+}
+
+// Topbar opening hours
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 14]
```diff
@@ -0,0 +1,438 @@
...
+  @include button(clock, $charcoal);
+  color:  $charcoal-opacity-extra-dark;
+ &:hover {
+    &::after {
+      color: $charcoal;
+    }
+  }
+}
+
+// Image caption
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 15]
```diff
@@ -0,0 +1,438 @@
...
+
+.field-image-caption {
+	 color: $white-opacity-dark;
+}	
+
+.ding-carousel {
+  .slick-slider {
+    .slick-track {
+      .placeholder {
+      background-image: none;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Indent is wrong.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 16]
```diff
@@ -0,0 +1,438 @@
...
+    &::after {
+      color: $charcoal;
+    }
+  }
+}
+
+// Image caption
+
+.field-image-caption {
+	 color: $white-opacity-dark;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Indent is wrong.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 17]
```diff
@@ -0,0 +1,438 @@
...
+      background-color: $bibliofil-dark;
+
+      .icon-spinner {
+        color: $charcoal-opacity-dark;
+      }
+      }
+    }
+  }
+}
+// Styling of Ding Tabroll
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot. Start with a blank line, remove blank line after.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 18]
```diff
@@ -0,0 +1,438 @@
...
+  .ui-tabs-nav {
+    .ui-tabs-nav-item {
+      a {
+        color: $white-opacity-dark;
+      }
+    }
+  }
+}
+
+// Ting reference
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 19]
```diff
@@ -0,0 +1,438 @@
...
+        input[type="submit"] {
+          background-color: $bibliofil-button;       
+        }
+      }
+    }
+  }
+
+   // Navigation
+  > .navigation-wrapper {
+      background-color: $color-topbar;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Wrong indent.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 20]
```diff
@@ -0,0 +1,438 @@
...
+  }
+
+  .navigation-inner {
+    background-color: $bibliofil-dark;
+  }
+}
+
+// Main menu
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 21]
```diff
@@ -0,0 +1,438 @@
...
+      a {
+        color: $white;
+        &::after {
+          background-color: $white;
+        }
+      }
+    }
+  }
+}
+// Main menu third level
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: End with a dot, start with a blank line.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 22]
```diff
@@ -0,0 +1,438 @@
...
+  > .inner {
+      > .buttons {
+        a {
+          color: $charcoal-opacity-dark;
+        }
+      }
+    }
+  }
+}
+//event
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 23]
```diff
@@ -0,0 +1,438 @@
...
+          .title-and-lead {
+          color: $white;
+          }
+        }
+      }
+    }
+  }
+}
+
+//news
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Start with a space, end with a dot.

File: themes/ddbasic/sass/bibliofil/_overides.scss [hunk 24]
```diff
@@ -0,0 +1,438 @@
...
+.pane-views-exp-ding-news-ding-news-list {
+  .views-exposed-form {
+    border-top: 1px solid $white-opacity-light;
+  }
+
+  .views-widget-filter-field_ding_news_category_tid,
+  .views-widget-filter-og_group_ref_target_id_entityreference_filter {
+    label {
+      h2 {
+         border-top: 1px solid $white-opacity-light;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_overides.scss]: Wrong indent.

File: themes/ddbasic/sass/bibliofil/_variables.scss [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+
+// ==========================================================================
+// Color
+// ==========================================================================
+
+$bibliofil: #4d4d51;
+$bibliofil-dark: #2c2c30;
+$bibliofil-button: #5b9f50;
+
+$black: #000;
```
Comment (guddo) [inline themes/ddbasic/sass/bibliofil/_variables.scss]: You set all variables again? Why not just set the ones needed?

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d81fc2f3ff73919c9cb85b2d118552fc8a1ba4b1
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: e96a23db44b9a8dc44440ef2cc41c3105947098f
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5930ce1ef5f34e671ece4b135925a126116da5e2
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 1b37dd83d191ba0fe18d425aced0620367bc911a
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 6ef7a95fcd9b352bd7f5255d70bd4da4d65b3e1b
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: fc14e85519016c9735641351de68a3ff066655e6
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15efcad94ac21b4459cdfae61ff80b76fc757dcd
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15efcad94ac21b4459cdfae61ff80b76fc757dcd
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15efcad94ac21b4459cdfae61ff80b76fc757dcd
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15efcad94ac21b4459cdfae61ff80b76fc757dcd
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 15efcad94ac21b4459cdfae61ff80b76fc757dcd
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b885aa4e956ca7b3b0b53f229f6cd3a3df1cc645
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 3fd133e3ec169f640dc3867f66a6671e70825543
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 61d1f5d0a4c920b1c34f53fa06892cc73cd89838
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: dbbf3eb1cef2b92fce7476047b1c5e343862b9ce
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 930edbce00ad528d9d24bceefeb3c955c9a0987f
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 4abc34b08cb5260191c0d2591977bc6140048c03
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b885aa4e956ca7b3b0b53f229f6cd3a3df1cc645
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 3fd133e3ec169f640dc3867f66a6671e70825543
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 61d1f5d0a4c920b1c34f53fa06892cc73cd89838
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: dbbf3eb1cef2b92fce7476047b1c5e343862b9ce
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 930edbce00ad528d9d24bceefeb3c955c9a0987f
  * Commits must be in the following format: [issue ID] - [short summary of change]

 * Commit: 4abc34b08cb5260191c0d2591977bc6140048c03
  * Your subject line is longer than 72 characters

 * Commit: 0114b7928bd492bc0885a98393903957509c6d1e
  * Commits must be in the following format: [issue ID] - [short summary of change]

Guidelines are available at https://github.com/easySuite/ding2

---
This message was auto-generated by https://gitcop.com

**Pull request closed without merging**